libs/template/experimental/if/src/lib/if.directive.ts

```diff
@@ -48,21 +48,20 @@ export class RxIf<U> implements OnInit, OnDestroy, OnChanges {
     rxIfTemplateNames
   >;
 
-  @Input()
-  set rxIf(potentialObservable: Observable<U> | U | null | undefined) {
-    this.observablesHandler.next(coerceObservable(potentialObservable));
-  }
+  private lastValidValue: U | null;
```

Comment:
General thought, what about using the private instance fields syntax? So we make sure these fields are not accessible at runtime.

```suggestion
  #lastValidValue: U | null;
```

Reply:
If we go that route I think we should do it in a separate PR for all of our directives/components